antarest/study/service.py

```diff
@@ -265,6 +274,58 @@ def create_study(
         )
         return str(raw.id)
 
+    def create_variant_study(
+        self, uuid: str, name: str, params: RequestParameters
+    ) -> Optional[str]:
+        """
+        Create empty study
+        Args:
+            uuid: study name to set
+            name: name of study
+            params: request parameters
+
+        Returns: new study uuid
+
+        """
+        study = self._get_study(uuid)
+        self._assert_permission(params.user, study, StudyPermissionType.READ)
+        if study is not None:
+            new_id = str(uuid4())
+            study_path = str(
+                self.raw_study_service.get_default_workspace_path() / new_id
+            )
+            variant_study = VariantStudy(
+                id=new_id,
+                name=name,
+                parent_id=uuid,
+                workspace=study.workspace,
+                path=study_path,
+                public_mode=study.public_mode,
+                created_at=datetime.now(),
+                updated_at=datetime.now(),
+                version=study.version,
+                type=study.type,
```

Comment:
the type here is "variant_study_snapshot" (well you don't need to specify it anyway since creating a VariantStudy automatically set this field to the correct value)